rune/libenclave/internal/runtime/pal/api_linux_v1.go

```diff
@@ -8,15 +8,19 @@ static int palGetVersion(void *sym)
 	return ((int (*)(void))sym)();
 }
 
-static int palInitV1(void *sym, const char *args, const char *log_level)
+static int palInitV1(void *sym, const char *args, const char *log_level, long addr, int fd)
```

Comment:
v1 is already released. v3 is under development so change to use v3 instead.

Reply:
Ok. Got it. Thanks